packages/nimble-components/src/wafer-map/index.ts

```diff
@@ -0,0 +1,64 @@
+import { attr } from '@microsoft/fast-element';
+import { DesignSystem, FoundationElement } from '@microsoft/fast-foundation';
+import { template } from '../theme-provider/template';
+import { styles } from './styles';
+import {
+    WaferMapColorBy,
+    WaferMapColorsScale,
+    WaferMapDataType,
+    WaferMapDie,
+    WaferMapOrientation,
+    WaferMapQuadrant
+} from './types';
+
+declare global {
+    interface HTMLElementTagNameMap {
+        'nimble-wafer-map': WaferMap;
+    }
+}
+
+/**
+ * A nimble-styled WaferMap
+ */
+export class WaferMap extends FoundationElement {
+    @attr({
+        attribute: 'wafer-quadrant'
+    })
+    public waferQuadrant!: WaferMapQuadrant;
+
+    @attr({
+        attribute: 'wafer-orientation'
+    })
+    public waferOrientation!: WaferMapOrientation;
+
+    @attr({
+        attribute: 'max-characters'
+    })
+    public maxCharacters!: number;
+
+    @attr({
+        attribute: 'wafer-datatype'
+    })
+    public waferDataType!: WaferMapDataType;
+
+    @attr({
+        attribute: 'colorBy'
+    })
+    public colorBy: WaferMapColorBy = WaferMapColorBy.hardBin;
+
+    @attr({
+        attribute: 'highlighted-values'
+    })
+    public highlightedValues!: string[];
+
+    public dice!: WaferMapDie[];
+    public colorsScale!: WaferMapColorsScale;
```

Comment:
One more thing - I would recommend to consider making this value an observable. I assume that this property is related to the "Zoom" feature. It is expected therefore that it will change based on user interaction, after the wafer-map was loaded ... therefore we would need to handle it in a callback. 

![image](https://user-images.githubusercontent.com/35329597/202733659-b699a192-6b6a-4402-8027-054b8d925171.png)